package/src/components/Attachment/Card.tsx

```diff
@@ -105,6 +106,8 @@ export type CardPropsWithContext<
     MessagesContextValue<At, Ch, Co, Ev, Me, Re, Us>,
     'additionalTouchableProps' | 'CardCover' | 'CardFooter' | 'CardHeader'
   > & {
+    channelId: string | undefined;
```

Comment:
maybe use optional parameter here

Reply:
Did some testing and having `string | undefined` makes passing the parameter mandatory which is what we want even if the value is undefined for some reason. Please do not consider the other similar comments.